smoke-tests-otel-starter/src/main/resources/logback.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-
-  <appender name="OpenTelemetry"
-            class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender">
-  </appender>
-
-  <root level="INFO">
-    <appender-ref ref="OpenTelemetry"/>
-  </root>
-
-</configuration>
```

Comment:
can we test both with and without xml configuration? if not possible, I think it's ok as long as we test one way here and the other way in the smoke test

Reply:
A test is executed with an XML file [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/e448478f9f419d6870ac3853e2448be8658dc41b/instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/logging/LogbackAppenderTest.java#L49). [It checks that the OpenTelemetry appender is added only once.](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/e448478f9f419d6870ac3853e2448be8658dc41b/instrumentation/spring/spring-boot-autoconfigure/src/testLogbackAppender/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/logging/LogbackAppenderTest.java#L63).

The Logback XML file was removed from the OpenTelemetry starter smoke test.

You mean to test both configurations in the smoke tests?

Reply:
oh, sorry, I missed (again) that this was under smoke-test 😅